app/graphql/queries/base_query.rb

```diff
@@ -0,0 +1,4 @@
+module Queries
+  class BaseQuery < GraphQL::Schema::Resolver
```

Comment:
I think we're going to need to add the following extensions (see my updated listings branch): 
```
extend GraphQL::Schema::Member::HasFields   
extend GraphQL::Schema::Resolver::HasPayloadType
```
That will allow us to add fields to our queries files